src/Tribe/Aggregator/Page.php

```diff
@@ -317,10 +316,10 @@ public function register_menu_item() {
 	 */
 	public function template( $name, $data = array(), $echo = true ) {
 		// Clean this Variable
-		$name = array_map( 'sanitize_title_with_dashes', (array) explode( '/', $name ) );
+		$name = array_map( 'sanitize_title_with_dashes', (array) explode( DIRECTORY_SEPARATOR, $name ) );
```

Comment:
Since PHP will always, no matter the OS, read the `/` char as a directory separator, the purpose of this is to explode on directory separators that might be generated, or coded, using both *nix and Windows line separators.

Use `preg_split` here:
```suggestion
		$name = array_map( 'sanitize_title_with_dashes', (array) preg_split( '~\\/~', $name ) );
```